lib/addons/cloudwatch-logs/index.ts

```diff
@@ -0,0 +1,110 @@
+import { PolicyDocument } from 'aws-cdk-lib/aws-iam';
+import { Construct } from "constructs";
+import * as iam from "aws-cdk-lib/aws-iam";
+import { setPath } from "../../utils";
+import { HelmAddOn, HelmAddOnUserProps } from "../helm-addon";
+import { ClusterInfo, Values } from "../../spi/types";
+import { createNamespace } from "../../utils/namespace-utils";
+import { getCloudWatchLogsPolicyDocument } from "./iam-policy";
+
+/**
+ * Configuration options for the FluentBit add-on.
+ */
+export interface CloudWatchLogsAddonProps extends HelmAddOnUserProps {
+    /**
+     * Iam policies for the add-on.
+     */
+    policyDocumentProvider?: (partition: string) => PolicyDocument;
+
+    /**
+     * Create Namespace with the provided one (will not if namespace is kube-system)
+     */
+    createNamespace?: boolean
+
+    /**
+     * Name of the service account for fluent bit.
+     */
+    serviceAccountName?: string;
+
+    /**
+     * CloudWatch Log retention days
+     */
+    logRetentionDays?: number;
+}
+/**
+ * Default props for the add-on.
+ */
+const defaultProps: CloudWatchLogsAddonProps = {
+    name: 'aws-for-fluent-bit',
+    chart: 'aws-for-fluent-bit',
+    release: "blueprints-addon-aws-fluent-bit-for-cw",
+    version: '0.1.24',
+    repository: 'https://aws.github.io/eks-charts',
+    namespace: 'aws-for-fluent-bit',
+    createNamespace: true,
+    serviceAccountName: 'aws-fluent-bit-for-cw-sa',
+    logRetentionDays: 90,
+    values: {}
+};
+
+/**
+ * CloudWatchLogsAddon deploys FluentBit into an EKS cluster using the `aws-for-fluent-bit` Helm chart.
+ * https://github.com/aws/eks-charts/tree/master/stable/aws-for-fluent-bit
+ * 
+ */
+export class CloudWatchLogsAddon extends HelmAddOn {
```

Comment:
Is this add-on mutually exclusive with aws-fluentbit-addon? if so, we should add a check and fail fast.